auth_token/middleware.py

```diff
@@ -41,7 +41,8 @@ def process_request(self, request):
 
     def _update_token_and_cookie(self, request, response, max_age, expires):
         request.token.change_and_save(
-            expires_at=compute_authorization_token_expires_at(get_auth_token_age(request)),
+            expires_at=compute_authorization_token_expires_at(get_auth_token_age(
+                request, request.token.user, request.token.auth_slug)),
```

Comment:
the problem is this is User model

Reply:
I can't use reques.user, because that can be AnonymousUser and wouldn't solve the issue I'm trying to solve here

Reply:
yes this is ok for me